test/service/test_manifest.py

```diff
@@ -433,8 +433,8 @@ def test_manifest_zarr(self):
         ]
         for entry in expected:
             url = furl(entry['file_drs_uri'])
-            entry['file_drs_uri'] = url.set(scheme='drs',
-                                            host=config.drs_domain).url
+            host = config.drs_domain or config.api_lambda_domain('service')
```

Comment:
This is highly confusing. In reality, the manifests that we generate in `dev` and `prod` point at the respective TDR instances. Only because the tests still use the DSS plugin, does the manifest generated during tests refer to Azul's DRS proxy. It's relatively brittle and pointless to hard-code that assumption here for the sake of this particular test.

Instead of patching the expectation, patch the actual response. Drop the host name from the DRS URIs in the response, after making sure it is present and not empty.